SpatialGDK/Source/SpatialGDK/Public/Utils/OpCallbackTemplate.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include <WorkerSDK/improbable/c_worker.h>
+#include "OpCallbackTemplate.generated.h"
+
+UCLASS(abstract)
+class SPATIALGDK_API UOpCallbackTemplate : public UObject
+{
+	GENERATED_BODY()
+
+public:
+	UOpCallbackTemplate() = default;
+	~UOpCallbackTemplate() = default;
+
+	/**
+      * Called by SpatialDispatcher on initialization, this is to allow the user to write more useful callbacks. 
+	  * @param World the UWorld top level hierarchy object.
+	  */
+	void Init(UWorld* World_) {
```

Comment:
I think we normally do `InWorld` rather than `World_` to avoid the warning. What do think @Vatyx @m-samiec